worker/worker.go

```diff
@@ -46,63 +42,6 @@ chmod +x worker
 ./worker "$@"
 `)
 
-// NewGoCodePackage creates an IronWorker code package from a Go package.
-// The codeName is the name the code package will have have once it is uploaded.
-//
-// The packageArgs are equivalent to the args you would give to `go build`.
-//
-// If the packageArgs are a list of .go files, they will be treated as a list of
-// source files specifying a single Go package.
-//
-// Otherwise it's treated as a Go package name that will be searched in GOPATH,
-// and compiled.
-//
-// Please note that if you don't use `package main` for your Go package, this
-// function will not work as expected.
-func NewGoCodePackage(codeName string, packageArgs ...string) (code Code, err error) {
```

Comment:
ironcli replaces this and probably everyone else is using NG. So this just seems like unnecessary maintenance?